src/databases/ffp/avtffpFileFormat.C

```diff
@@ -14,7 +14,12 @@
 
 #include <avtffpFileFormat.h>
 
+#ifndef WIN32
```

Comment:
FWIW...I find negating logic a tad harder to read and so would code this a little differently...

```
#ifdef WIN32
#include <windows.h>
#else
#include <dlfcn.h>
#endif
```

Similarly for other `WIN32` specific code blocks.

Nonetheless, I don't mind it as is.